tests/test_constraint_manager.py

```diff
@@ -379,21 +591,48 @@ def _create_multi_model_both_constraints(self):
                 constraints:
                   perf_throughput:
                     min: 100
-              model_C:
+        """)
+        config = evaluate_mock_config(args, yaml_str, subcommand="profile")
+
+        return config
+
+    def _create_multi_model_with_global_constraints(self):
+        args = self._create_args()
+        yaml_str = ("""
+            profile_models:
+              model_A:
+                constraints:
+                  perf_latency_p99:
+                    max: 75
+              model_B:
                 constraints:
-                  gpu_used_memory:
-                    max: 50
                   perf_throughput:
-                    min: 50
-              model_D:
-                objectives:
-                  perf_throughput  
-                    
+                    min: 125
             constraints:
-                perf_latency_p99:
-                  max: 100
+                perf_throughput:
+                  min: 100
                 gpu_used_memory:
-                  max: 100
+                  max: 1000
+        """)
+        config = evaluate_mock_config(args, yaml_str, subcommand="profile")
+
+        return config
+
+    def _create_multi_model_with_different_all_constraints(self):
```

Comment:
change to with_different_global_constrants

Reply:
Done 👍